### PR TITLE
Create or update baselines to match 2.0

### PR DESCRIPTION
**Pull request by ryanbrandenburg**

### CODE REVIEW

**Comment by javiercn** [comment]
@ryanbrandenburg I think there might be libraries that are missing baselines in here. @HaoK

File: src/AspNetCoreCompat/baseline.netframework.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "AssemblyIdentity": "Microsoft.AspNet.Identity.AspNetCoreCompat, Version=0.1.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
+  "AssemblyIdentity": "Microsoft.AspNet.Identity.AspNetCoreCompat, Version=0.3.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60",
```
Comment (HaoK) [inline src/AspNetCoreCompat/baseline.netframework.json]: Do we need to baseline prerelease non shipping alphas?

**Comment by HaoK** [comment]
src/Core and src/Stores need baselines as well I think those are the main two that are missing baselines that shipped and are public in 2.0

**Comment by ryanbrandenburg** [comment]
:up::date:

**Pull request merged**